api/address/routes.go

```diff
@@ -23,6 +23,7 @@ const (
 	getESDTTokens   = "/:address/esdt"
 	getESDTBalance  = "/:address/esdt/:tokenIdentifier"
 	getESDTNFTData  = "/:address/esdtnft/:tokenIdentifier/nonce/:nonce"
```

Comment:
Maybe rename `esdtnft` to simply `nft`? The longer version would translate to Elrond Standard Digital Token Non Fungible Token

Reply:
done